.travis.yml

```diff
@@ -2,7 +2,7 @@ sudo: false
 
 language: go
 go:
-- 1.5
```

Comment:
With Travis, you can actually do `1.x` to resolve to the latest point release of Go, so in this case, 1.7.4.